Airwallex/Card/Internal/AWXCardViewModel.swift

```diff
@@ -0,0 +1,262 @@
+//
+//  AWXCardViewModel.swift
+//  Card
+//
+//  Created by Tony He (CTR) on 2024/7/25.
+//  Copyright © 2024 Airwallex. All rights reserved.
+//
+
+import Foundation
+
+@objc public protocol AWXCardviewModelDelegate {
+    @objc optional func startLoading()
+    @objc optional func stopLoading()
+    @objc func shouldDismiss(completeStatus status: AirwallexPaymentStatus, error: (any Error)?)
+    @objc optional func didCompleteWithPaymentConsentId(_ id: String)
+    @objc optional func shouldShowError(_ error: String)
+    @objc func shouldPresent(_ controller: UIViewController?, forceToDismiss: Bool, withAnimation: Bool)
+    @objc func shouldInsert(_ controller: UIViewController?)
+}
+
+@objcMembers
+@objc
+public class AWXCardViewModel: NSObject {
+    public weak var delegate: AWXCardviewModelDelegate?
+
+    public var ctaTitle: String {
+        if session is AWXRecurringSession {
+            return NSLocalizedString("Confirm", comment: "Confirm button title")
+        } else {
+            return NSLocalizedString("Pay", comment: "Pay button title")
+        }
+    }
+
+    private let pageName: String = "card_payment_view"
```

Comment:
Has this ever been used? It was used for page tracking, see AWXPageViewTrackable that AWXCardViewController used to conform to.